prime-router/metadata/fhir_mapping/hl7/datatype/Annotation.yml

```diff
@@ -36,7 +39,7 @@ extension_1:
       constants:
         system: v2-0105
 
-extension_2:
```

Comment:
In the NTE[ServiceRequest] mapping guide for Comment Type to extension.notetype in column P there's a link to a value set. Should that be accounted for in the mapping?

Reply:
Great call... this is definitely missing and does exist in the libraries code map.  Will update

Reply:
Actually, taking a closer look I don't think we need to anything special here since this is an extension and not mapping to a different valueset for FHIR.  The codeable concept we produce will include the code and display